config/remotePlugin.js

```diff
@@ -81,6 +81,10 @@ module.exports = {
       ComponentSettings: resolve(__dirname, '../src/pages/ComponentSettingsPage'),
       WorkspaceSettings: resolve(__dirname, '../src/pages/WorkspaceSettingsPage'),
       CreateEnvironment: resolve(__dirname, '../src/pages/CreateEnvironmentPage'),
+      NamespaceContext: resolve(
+        __dirname,
+        '../src/components/NamespacedPage/namespace-context-provider',
```

Comment:
I think we ought to move `useNamespace` and this context to the utils dir and out of the NamespacedPage component / dir.